charts_flutter/example/lib/bar_chart/bar_gallery.dart

```diff
@@ -39,6 +39,25 @@ import 'vertical_bar_label.dart';
 const simpleBarChartTileTitle = 'Simple Bar Chart';
 const stackedBarChartTileTitle = 'Stacked Bar Chart';
 const groupedBarChartTileTitle = 'Grouped Bar Chart';
+const groupedStackedBarChartTileTitle = 'Grouped Stacked Bar Chart';
+const groupedBarTargetLineChartTileTitle = 'Grouped Bar Target Line Chart';
+const groupedBarSingleTargetLineChartTileTitle =
+    'Grouped Bar Single Target Line Chart';
+const stackedBarTargetLineChart = 'Stacked Bar Target Line Chart';
+const horizontalBarChart = 'Horizontal Bar Chart';
+const stackedHorizontalBarChart = 'Stacked Horizontal Bar Chart';
+const horizontalBarLabelChart = 'Horizontal Bar Chart with Bar Labels';
+const horizontalBarLabelCustomChart =
+    'Horizontal Bar Chart with Custom Bar Labels';
+const verticalBarLabelChart = 'Vertical Bar Chart with Bar Labels';
+const sparkBarChart = 'Spark Bar Chart';
+const groupedFillColorChart = 'Grouped Fill Color Bar Chart';
+const stackedFillColorChart = 'Stacked Fill Color Bar Chart';
+const patternForwardHatchChart = 'Pattern Forward Hatch Bar Chart';
+const horizontalPatternForwardHatchChart =
+    'Horizontal Pattern Forward Hatch Bar Chart';
+const weightedPatternChart = 'Weighted Pattern Bar Chart';
```

Comment:
@abdushakoor12 Goldens FTW!

When I generated the goldens there was a slight difference for the menu golden. I couldn;t figure out what it was, so I got Claude to analyze the difference between the two files. It found that this text should be: `Weight Pattern Bar Chart` and not `Weighted Pattern Bar Chart`

<img width="515" alt="image" src="https://github.com/user-attachments/assets/5954007c-d287-41e9-9b44-54681cc44e4f">

Reply:
@abdushakoor12 it turns out that there were more. I fixed it all up in this [PR](https://github.com/Nimblesite/nimble_charts/pull/15/files)

Reply:
@MelbourneDeveloper  Oh nice, like what you did with tileFinder predicate.